mods/example/mod.yaml

```
@@ -66,12 +67,18 @@ MapGrid:
 
 SpriteFormats: ShpTS
 
```

```suggestion

SoundFormats: Wav
```

as https://github.com/OpenRA/OpenRAModSDK/blob/e0c1ee07e4589335e92c8142e5f29530d4b4c6a5/mods/example/audio/notifications.yaml#L2

already mentions it.

---

Ok.